ledger/acctupdates.go

```diff
@@ -1172,7 +1172,7 @@ func (au *accountUpdates) lookupLatest(addr basics.Address) (data basics.Account
 
 		if resourceDbRound < currentDbRound {
 			au.log.Errorf("accountUpdates.lookupLatest: resource database round %d is behind in-memory round %d", resourceDbRound, currentDbRound)
-			return basics.AccountData{}, basics.Round(0), basics.MicroAlgos{}, &StaleDatabaseRoundError{databaseRound: resourceDbRound, memoryRound: currentDbRound}
+			/*xxxxx*/ return basics.AccountData{}, basics.Round(0), basics.MicroAlgos{}, &StaleDatabaseRoundError{databaseRound: resourceDbRound, memoryRound: currentDbRound}
```

Comment:
This line will be hit from line 2446 in TestAcctUpdatesLookupLatestCacheRetry and the test will fail.
Investigating what is causing this failure, and what needs to be fixed (prod code or test).

Reply:
moved to https://github.com/algorand/go-algorand/pull/5566